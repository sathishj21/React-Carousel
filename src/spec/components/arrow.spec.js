import React from 'react';
import { shallow } from 'enzyme';
import Arrow from '../../components/arrow';

const defaultProps = {
    id: "prev",
    clickAction:jest.fn(),
    content: "Prev"
};


it('shallow renders Carousel component without crashing', () => {
    shallow(<Arrow {...defaultProps} />);
});