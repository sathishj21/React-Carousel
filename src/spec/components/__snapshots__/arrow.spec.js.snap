// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shallow renders Carousel component without crashing 1`] = `
<button
  className="button-left"
  id="prev"
  onClick={[MockFunction]}
>
  <span
    className="icon icon-arrow-left"
  />
  <span
    className="button-content"
  >
    Prev
  </span>
</button>
`;
